Cargo.toml

```diff
@@ -9,9 +9,11 @@ repository = "https://github.com/cloudhead/nonempty"
 
 [dependencies]
 serde = { features = ["serde_derive"], optional = true, version = "1" }
+arbitrary = { features = ["derive"], optional = true, version = "1" }
```

Comment:
I don't think we need `derive` feature

Reply:
That is specifically used for the test case, to provide a usage example. I pull in the derive feature out of symmetry with the feature above that pulls in the `serde` dependency with `serde_derive`. 

I believe I can remove it though, as it's not strictly required, but then an argument could be made that the above `features =` needs to go as well. 

Since this is not my codebase, I valued symmetry over my own opinions.

Reply:
@FintanH Do you have any strong opinion on the above ?

Reply:
Ended up removing the features after removing the newtype in the tests as it wasnt necessary.

Reply:
Sweet, ya I think it makes sense not to have it if it's not needed :+1:

Reply:
Note: I reverted to using the proc-macro everywhere for easier maintainance, so we actually do need that feature now